tests/test_rdf_parser.py

```diff
@@ -42,6 +43,6 @@ def check_document(self, document, expected_loc, regen=False):
                 o.write(data)
 
         with io.open(expected_loc, 'r', encoding='utf-8') as ex:
-            expected = json.load(ex)
+            expected = json.load(ex, object_pairs_hook=OrderedDict)
```

Comment:
There is a weird type warning on this line, but it appears to do what we want, so I guess it's fine?